### PR TITLE
Ports Improved Parallax

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Ports @Qustinnus improvements to parallax.

This adds an additional parallax layer to parallax that's randomized at shift start---the layer could be completely absent, or it could be asteroids or even space gas---if it's space gas, the color will be randomized.

This also makes Lavaland bigger, in the background, since, well, the station is orbiting the planet and it's difficult to notice in the current iteration, anyway.

Some pics:

img
img

:cl: Fox McCloud
add: Adds an additional parallax layer that is randomized; it may be asteroids, randomly colored space gas, or not even there at all
tweak: Lavaland parallax is bigger; the station is orbiting the planet, afterall
/:cl:

### CODE REVIEW

**Comment by Denthamos** [comment]
In the words of the space core. "_SpaaaaaaaaAAAAaaaaaaaace"_

**Comment by S34NW** [comment]
looks really smart

**Review by SteelSlayer — APPROVED** [review]
Not an expert on parallax, but I tested several rounds and everything works fine, so LGTM.

**Comment by KasparoVy** [comment]
Seeing such a drop-dead gorgeous asteroid field parallax makes me wish that's where mining was. Gah, bloody hell-planet hahaha

**Review by dearmochi — APPROVED** [review]

**Review by Kyep — APPROVED** [review]
I'm not sure about this, since it gives more prominence to lavaland (lavaland already gets way too much special treatment), and some of the random colors of space gas could potentially look bad.
However, I'm willing to give this a shot.

File: code/controllers/subsystem/parallax.dm [hunk 1]
```diff
@@ -8,12 +8,20 @@ SUBSYSTEM_DEF(parallax)
 	var/list/currentrun
 	var/planet_x_offset = 128
 	var/planet_y_offset = 128
+	var/random_layer
+	var/random_parallax_color
 
-/datum/controller/subsystem/parallax/Initialize(timeofday)
```
Comment (AffectedArc07) [inline code/controllers/subsystem/parallax.dm:12]: If were doing this, add `SS_NO_INIT` to the flags

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**